packages/template-retail-react-app/config/default.js

```diff
@@ -36,6 +36,12 @@ module.exports = {
             // This differs from the siteId in commerceAPIConfig for testing purposes
             siteId: 'aaij-MobileFirst',
             isProduction: false
+        },
+        // getAppOrigin will take x-forwarded-host header into account if this is true
+        useXForwardedHost: false,
+        login: {
+            allowPasswordless: false,
+            idps: []
```

Comment:
I like that you are thinking about how the configuration for login will work holistically. I'd like to see some more options on how to organize this tho. 

My primary feedback on this current iteration is that `idps` is kinda there on it's own, I'm assuming that it's related to social login feature?

Personally I'm a fan of overloading configurations like this:

```
login: {
   passwordless: true,
   social: {
       enabled: true
    }
}
```

and also allow:

```
login: {
   passwordless: {
       enabled: true,
       redirectUrl: ''
    },
   social: {
       enabled: true
    }
}
```

You can see above that for passwordless you are able to configure the callback url, because customers might just want to do that, but it its not provided we use the default.

Reply:
Great point will update so there's another level so we can add further configurations

Reply:
The only reason I like an explicit "enabled or disabled" property is that you don't have to delete a configuration if you are debugging something and simply want to turn idp's off. I suppose that can be added later tho if required

Reply:
Ooh thats a great point. Will add it since its pretty simple